prow/kube/prowjob.go

```diff
@@ -124,6 +138,19 @@ type ProwJobSpec struct {
 	RunAfterSuccess []ProwJobSpec `json:"run_after_success,omitempty"`
 }
 
+// UtilityImages holds pull specs for the utility images
+// to be used for a job
+type UtilityImages struct {
+	// CloneRefs is the pull spec used for the clonerefs utility
```

Comment:
I was interested in seeing godoc specific to each field. For example:
```go
// CloneRefs is the image used for cloning all the necessary source code
// inside the test container. Read more about clonerefs in <link to clonerefs readme>
```

Reply:
And the readme should contain info on how to use the utility manually and a reference to how prow is using it. Addings READMEs is not something that should be handled here tho.